pkg/proxy/winkernel/proxier.go

```diff
@@ -658,6 +667,38 @@ func NewProxier(
 
 }
 
+func NewDualStackProxier(
+	syncPeriod time.Duration,
+	minSyncPeriod time.Duration,
+	masqueradeAll bool,
+	masqueradeBit int,
+	clusterCIDR string,
+	hostname string,
+	nodeIP [2]net.IP,
+	recorder record.EventRecorder,
+	healthzServer healthcheck.ProxierHealthUpdater,
+	config config.KubeProxyWinkernelConfiguration,
+) (proxy.Provider, error) {
+
+	// Create an ipv4 instance of the single-stack proxier
+	ipv4Proxier, err := NewProxier(syncPeriod, minSyncPeriod, masqueradeAll, masqueradeBit,
+		clusterCIDR, hostname, nodeIP[0], recorder, healthzServer, config)
+
+	if err != nil {
+		return nil, fmt.Errorf("unable to create ipv4 proxier: %v", err)
```

Comment:
Do you mind adding some additional contextual information (host name, cluster cidr, node IP, ...). This greatly helps when debugging at scale.